### DISCUSSION TITLE
TerriaMap - Tabs in the data catalogs in Terria v8

### DISCUSSION THREAD

**Discussion by LorenzoStucchi** [TerriaJS/terriajs#5768] [Q&A] [answered]
Hi, 
I'm testing the new version of Terria into TerriaMap, I used the tool to convert the file in the `init` folder. However, I'm not anymore able to create different tabs for the division of the data as previously. Is this a feature that has been removed?

Here the example:
The visualization in v7
![v7](https://user-images.githubusercontent.com/29042221/131092812-1869041f-a8a4-4772-a65b-fcd834eeeaba.png)

And now in v8 
![v8](https://user-images.githubusercontent.com/29042221/131092855-b7310549-8a14-4b7f-bfba-1e0616d97927.png)

**Comment by hardreddata** [comment, marked as answer]
Have you set `"tabbedCatalog": true,` in `config.json`?

**Reply by LorenzoStucchi** [reply]
Thanks, it was not copied and inserted correctly 😄
